application/src/main/java/org/thingsboard/server/service/security/auth/TokenOutdatingService.java

```diff
@@ -17,65 +17,44 @@
 
 import io.jsonwebtoken.Claims;
 import lombok.RequiredArgsConstructor;
-import org.springframework.cache.Cache;
-import org.springframework.cache.CacheManager;
 import org.springframework.context.event.EventListener;
 import org.springframework.stereotype.Service;
-import org.thingsboard.server.common.data.CacheConstants;
+import org.springframework.util.StringUtils;
```

Comment:
We don't use StringUtils from Spring. We use our own StringUtils. Consider adding a method there if you need it.